pages/login.html

```diff
@@ -0,0 +1,148 @@
+          <input
+            type="text"
+            class="search-input w-100 p-2 m-2 text-sm"
+            placeholder="Search for products..."
+            required
+          />
+          <button type="submit" class="btn btn-icon p-4">
+            <i class="fas fa-search text-md"></i>
+          </button>
+        </div>
+        <ul class="nav-links d-flex align-center">
+          <li>
+            <button
+              type="button"
+              class="btn btn-primary rounded-sm text-sm p-4"
+            >
+              Signup
+            </button>
+          </li>
+          <li class="p-relative text-md icon-badge-wrapper m-3">
+            <i class="fas fa-shopping-cart"></i>
+            <span
+              class="badge icon-badge-position text-sm font-wt-bold rounded-full p-absolute"
+              >5</span
+            >
+          </li>
+          <li class="p-relative text-md icon-badge-wrapper m-3">
+            <i class="fas fa-heart"></i>
+            <span
+              class="badge icon-badge-position text-sm font-wt-bold rounded-full p-absolute"
+              >7</span
+            >
+          </li>
+        </ul>
+      </header>
+
+      <!-- login card -->
+      <section class="login-card-wrapper container-center">
+        <div class="login-card rounded-md box-shadow-sm">
+          <div class="d-flex flex-col px-8 py-4">
+            <label class="input-label-required text-sm my-2" for="email"
+              >Email</label
+            >
+            <input
+              type="text"
+              class="text-input w-100 p-3 text-sm"
+              placeholder="abc@gmail.com"
+              required
+            />
+          </div>
+
+          <div class="d-flex flex-col px-8 py-4">
+            <label class="input-label-required text-sm my-2" for="password"
+              >Password</label
+            >
+            <input
+              type="password"
+              class="text-input w-100 p-3 text-sm"
+              placeholder="Enter password here"
+              required
+            />
+          </div>
+          <div class="d-flex align-center px-8 py-4">
+            <input
+              type="checkbox"
+              id="remember-password"
+              name="remember-password"
+            />
+            <label class="text-sm ml-2" for="remember-password">
+              Remember Password</label
+            >
+            <div class="text-sm ml-auto">
+              <a class="link-text" href="#">Forgot your Password ?</a>
+            </div>
+          </div>
+          <div class="px-8 py-4">
+            <button
+              type="button"
+              class="btn btn-primary rounded-sm text-sm p-4 w-100"
+            >
+              Login
+            </button>
+
+            <div class="text-sm ml-auto d-flex justify-center">
+              <a class="link-text" href="#"
+                >Already have an Account
+                <i class="fas fa-greater-than ml-2 my-4"></i
```

Comment:
I don't think you'll need this as it is generally on signup page not login page

Reply:
oh yes ... my bad. I will correct it now.